Index_2.html

```diff
@@ -0,0 +1,60 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="Ervin Besic">
+    <meta name="description" content="A book about markup languages">
+    <meta http-equiv="refresh" content="40">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>A book about markup languages</title>
+</head>
+
+<body>
+
+    <h1>1. Markup languages</h1>
+    <p>A markup language is a computer language that uses tags to define elements within a document.<br>
+        Markup languages are used to annotate a document in a way that is syntactically distinguishable from the
+        text,<br>
+        and can assign semantic meaning to the text.<br>
+        <br><br>
+        <!--Posebno oznacen text-->
+        <strong>Types of markup languages</strong>
+        <ul>

```

Comment:
Popisi se ne mogu nalaziti unutar paragrafa.

`<p>` element smije sadržavati samo elemente iz [_phrasing content_](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content) skupine. To su generalno elementi kojima uređujemo pojedine riječi u tekstu, i slično. Više o tome možeš pročitati [ovdje](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p).

`<ul>` element (i ostale liste) pripada [_flow content_](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Flow_content) skupini elemenata, zbog čega ne može biti smješten unutar paragrafa. Više o tome možeš pročitati [ovdje](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul).